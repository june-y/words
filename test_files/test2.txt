 juvenile,nectar,gold
R	A	T	C	E	N	T	J
O	O	Z	K	E	E	U	Z
B	M	R	K	D	V	G	Q
D	L	E	P	E	A	R	D
D	Y	P	N	L	J	S	D
L	L	I	B	D	S	O	G
W	L	O	T	Y	X	Y	S
E	N	C	G	V	A	I	V




