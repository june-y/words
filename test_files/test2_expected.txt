juvenile:
nectar:
gold: (7,3) (6,2) (5,1) (4,0)
 





R[0,0]	A[1,0]	T	C	E	N	T	J[7,0]
O[0,1]	O	Z	K	E	E	U	Z
B[0,2]	M	R	K	D	V	G	Q
D	L	E	P	E[]	A	R	D
D	Y	P	N	L	J	S	D
L	L	I	B	D	S[]	O	G
W	L	O	T	Y	X	Y	S
E[0,7]	N	C	G	V	A	I	V[7,7]





